src/pybind/mgr/cephadm/module.py

```diff
@@ -2193,6 +2197,27 @@ def _refresh_configs(self):
                 self._create_daemon(dd.daemon_type, dd.daemon_id,
                                     dd.hostname, reconfig=True)
 
+        # make sure the dashboard [does not] references grafana
+        try:
+            current_url = self.remote('dashboard', 'get_option',
+                                      'GRAFANA_API_URL')
+            current_ssh_verify = self.remote('dashboard', 'get_option',
```

Comment:
This line isn't used. If we want to keep it around as a reminder, we should probably have it named `current_ssl_verify`.